extensions/amp-base-carousel/0.1/carousel.js

```diff
@@ -733,7 +734,7 @@ export class Carousel {
         dict({
           'index': index,
           'total': this.slides_.length,
-          'offset': offset,
+          'offset': this.forwards_ ? -offset : offset,
```

Comment:
Looks like we keep using this as a direction, maybe switch the variable to `this.direction_` (-1 or 1) instead?

Reply:
Direction can be a little confusing, since we are dealing with flex direction in layout, which can be `row`, `row-reverse`, `column` and `column-reverse`. I chose `forwards` since it is the opposite of the reverse flex directions, independent of the axis.